config/lobby/db/004_audit_bans

```diff
@@ -0,0 +1,23 @@
+start transaction;
+
+alter table banned_macs
+  add column mod_username varchar(40),
+  add column mod_ip varchar(45),
+  add column mod_mac char(28),
```

Comment:
`mac` is not secure/reliable - can be manipulated by a patched client. My intent was to remove mac and all tables to simplify. Given that, would it make sense to go ahead and remvoe it? Is there a case to keep MAC around?

Reply:
I misunderstood the columns, it's not the bannee creds, but the banner.

Reply:
I can change the `mod_` prefix to `banner_` if you think that would help.